### PR TITLE
Download source code from npm over github if possible.

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

**Review by chee — APPROVED** [review]
wooooo

looking swell

File: lib/download-repo.js [hunk 1]
```diff
@@ -1,37 +1,84 @@
...
+/**
+ * @param {string} npmTarballURL - url to download tarball from
+ * @param {number} tarballRequestTimeout - npm request timeout in milliseconds
+ * @returns {stream.Writable}
+ */
+function getNpmTarballStream(npmTarballURL, tarballRequestTimeout) {
+    // Validate the npm host is as expected, this is generated by a 3rd-party
+    // library at the time of writing.
+    const expectedNpmHost = 'registry.npmjs.org';
+    if (new URL(npmTarballURL).host !== expectedNpmHost) {
```
Comment (JakeChampion) [inline lib/download-repo.js:43]: A future piece of work would be to allow this to download from our staging npm registry

Comment (notlee) [reply lib/download-repo.js:43]: Or, a future piece of work would be to delete this project! :D Maybe whatever building the new site/registry could also build codedocs. We could look to build a proper Origami theme for sassdoc/jsdoc, which others could use, or drop the theme entirely depending on what users think...

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v1.0.6 :tada:

**Pull request merged**